Demo/Sources/Settings/SettingsView.swift

```diff
@@ -8,6 +8,35 @@ import AVFoundation
 import Player
 import SwiftUI
 
+private struct UrlCacheView: View {
+    @State private var urlCacheSize: String = ""
+
+    var body: some View {
+        HStack {
+            Button("Clear URL cache", action: clearUrlCache)
+#if os(iOS)
+                .buttonStyle(.borderless)
+#endif
+            Spacer()
+            Text(urlCacheSize)
+                .font(.footnote)
+                .foregroundColor(.red)
+        }
+        .onAppear {
+            updateCacheSize()
+        }
```

Comment:
The `currentDiskUsage` property seems to be KVObservable. You can therefore likely use `.onReceive(URLCache.shared.publisher(for: \.currentDiskUsage))` to not only get the initial value, but also potential updates which could happen while the view is visible on screen.

Reply:
Thank you for your suggestion. It was the first thing I attempted. Unfortunately, it didn't work. I'll give it another try and inform you of the results.

Reply:
You're right. The current implementation is good enough, then, thanks.